### ISSUE TITLE
KeepassXC crashes with grayed window after merge and save

### ISSUE THREAD

**Issue by delize** [keepassxreboot/keepassxc#4454]
After saving the database on google drive file stream (offline copy) on computer 1, and then switching to computer 2 and making an update/modification, the database will attempt to save, and then stay grayed out with no response. 

When you try to quit the app, the database will stay open, and it is required to quit the app via Task Manager or Force Quit via macOS.

This setup that I am experiencing this on is 
2x Windows 10 Pro x64 & macOS 10.15 & macOS 10.15.3

## Expected Behavior
When you save, the app should merge the previous database changes, and then save overtop those changes merging all changes upstream.

## Current Behavior
Database tries to save, grays out the entire screen, and becomes unresponsive. Requiring task manager or app activity to force quit the application.

## Possible Solution
None available that I am aware of

## Steps to Reproduce
1.  Save database on computer 1 with changes on Drive File Stream
2. Wait a few hours, switch to computer 2
3. Save a new or existing updated record on computer 2
4. Computer 2's Keepass instance will turn gray and become unresponsive.

## Context
* Google Drive File Stream
* Mixed environment of macOS and Windows
* Up to 4 devices using the database at a given time

## Debug Info
KeePassXC - 2.5.3
Revision: f8c962b

Windows & macOS:
```
KeePassXC - Version 2.5.3
Revision: f8c962b

Qt 5.13.2
Debugging mode is disabled.

Operating system: Windows 10 (10.0)
CPU architecture: x86_64
Kernel: winnt 10.0.18363

Operating system: macOS 10.15(.3)
CPU architecture: x86_64
Kernel: darwin 19.3.0

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey

Cryptographic libraries:
 libgcrypt 1.8.5


Libraries:
- LIBS
```

**Comment by droidmonkey** [comment]
Can you try using our snapshot build. I believe to have fixed this issue before (this is a duplicate report).

https://snapshot.keepassxc.org

**Comment by ctand** [comment]
This is not fixed in the 2.6 snapshot and was introduced in 2.5.3. I believe it to be related to #4466 as well.

**Comment by droidmonkey** [comment]
This is a duplicate
